projects/fountain.js

```diff
@@ -1,15 +1,6 @@
 const { compoundExports } = require("./helper/compound");
+const { transformOasisAddressBase } = require('./helper/portedTokens')
 
-function transformOasis(addr) {
-    const map = {
-        '0x21c718c22d52d0f3a789b752d4c2fd5908a8a733':'oasis-network',
-        '0x3223f17957ba502cbe71401d55a0db26e5f7c68f':'0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
-        '0xdc19a122e268128b5ee20366299fc7b5b199c8e3':'0xdac17f958d2ee523a2206206994597c13d831ec7',
-        '0xe8a638b3b7565ee7c5eb9755e58552afc87b94dd': '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
-    }
-    return map[addr.toLowerCase()] || `oasis:${addr}`
-
-}
 module.exports = {
     timetravel: true,
     aurora: {
```

Comment:
wait, should this chain be changed to oasis?

Reply:
It looks like this adapter code isnt actually used on our front end